test/fluture.test.js

```diff
@@ -6,11 +6,11 @@ const Readable = require('stream').Readable;
 const jsc = require('jsverify');
 const S = require('sanctuary');
 const FL = {
-  map: 'map',
-  bimap: 'bimap',
-  chain: 'chain',
-  ap: 'ap',
-  of: 'of'
+  map: 'fantasy-land/map',
+  bimap: 'fantasy-land/bimap',
+  chain: 'fantasy-land/chain',
+  ap: 'fantasy-land/ap',
+  of: 'fantasy-land/of'
 };
```

Comment:
> FantasyLand comes with its own dependency tree which seems to me like unnecessary overhead.

Didn't know that, created a PR that fixes that https://github.com/fantasyland/fantasy-land/pull/168

Other points are valid though, I understand it better now, thanks!